src/lib/containers/download_list_v2/DownloadListTable.tsx

```diff
@@ -190,7 +193,7 @@ export default function DownloadListTable() {
                   const createdOn = moment(item.createdOn).format('L LT')
                   return (
                     <tr key={item.fileEntityId}>
-                      <td>
+                      <td className={item.isEligibleForPackaging ? '' : 'ineligibleForPackagingTd'}>
```

Comment:
Special css class if item is ineligible for packaging (has a special background color)